sycl/test-e2e/KernelFusion/sync_acc_mem_op.cpp

```diff
@@ -1,6 +1,5 @@
 // RUN: %{build} -fsycl-embed-ir -o %t.out
 // RUN: env SYCL_RT_WARNING_LEVEL=1 %{run} %t.out 2>&1 | FileCheck %s
-// UNSUPPORTED: hip
```

Comment:
This was intentional, yes. 

These tests do not need the actual JIT compiler, because fusion is always cancelled due to some synchronization event (that's what the tests check). 
As the API itself is not hidden if the feature is unavailable, these tests were meant to at least check no deadlocks or similar would occur. 

On the other hand, it's certainly debatable whether there's value in testing a scenario where the user ignored the fact that a feature is unavailable and still uses its API.

Reply:
> Was it intentional? If so, I should abandon this PR.

I can still move 'UNSUPPORTED: hip" to the `lit.local.cfg`. Doing it now.